src/main/java/gregtech/common/gui/widget/craftingstation/ItemListSlotWidget.java

```diff
@@ -194,7 +193,7 @@ public void handleClientAction(int id, PacketBuffer buffer) {
     private void dispatchMouseClick(int button) {
         IItemInfo itemInfo = gridWidget.getItemInfoAt(index);
         handleMouseClick(itemInfo, button, true);
-        ItemStack itemStack = itemInfo == null ? ItemStack.EMPTY : itemInfo.getItemStackKey().getItemStack();
+        ItemStack itemStack = itemInfo == null ? ItemStack.EMPTY : itemInfo.getItemStack();
```

Comment:
per your migrating guide, shouldn't this be a copy?

Reply:
Since this stack is only used for writing to a packet, and is not stored for further modification elsewhere, it does not need to be copied.